src/ogt_vox.h

```diff
@@ -318,11 +324,15 @@
 //
 //-----------------------------------------------------------------------------------------------------------------
 #ifdef OGT_VOX_IMPLEMENTATION
+#ifndef ogt_assert
     #include <assert.h>
+    #define ogt_assert(x, msg) assert(x)
```

Comment:
Perhaps we should document the ability to override assert in the non-implementation section via a comment?